pkg/orm/orm.go

```diff
@@ -85,6 +85,8 @@ var (
 	ErrStmtClosed    = errors.New("<QuerySeter> stmt already closed")
 	ErrArgs          = errors.New("<Ormer> args error may be empty")
 	ErrNotImplement  = errors.New("have not implement")
+
+	ErrLastInsertIdUnavailable = errors.New("<Ormer> last insert id is unavailable")
```

Comment:
If we define this error, why not just return this error so that Users can check this error. It looks like:
```golang
 err := InsertOrUpdate(xxx)
if err == ErrLastInsertIdUnavailable {
    // do nothing
} else {
   // really failed.
}
```